Library/Formula/libgraphqlparser.rb

```diff
@@ -0,0 +1,203 @@
+require "open3"
```

Comment:
To be a little annoying, if both of these are required only for the `test do` stage, let's move them to the top of that block.